<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>Sidebar</class>
	<widget class="QWidget" name="Sidebar">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>252</width>
				<height>800</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Sidebar</string>
		</property>
		<property name="styleSheet">
			<string notr="true"></string>
		</property>
		<layout class="QVBoxLayout" name="main_layout">
			<property name="spacing">
				<number>0</number>
			</property>
			<property name="leftMargin">
				<number>0</number>
			</property>
			<property name="topMargin">
				<number>0</number>
			</property>
			<property name="rightMargin">
				<number>0</number>
			</property>
			<property name="bottomMargin">
				<number>0</number>
			</property>
			<item>
				<widget class="QWidget" name="header_widget" native="true">
					<property name="sizePolicy">
						<sizepolicy hsizetype="Preferred" vsizetype="Fixed">
							<horstretch>0</horstretch>
							<verstretch>0</verstretch>
						</sizepolicy>
					</property>
					<property name="minimumSize">
						<size>
							<width>0</width>
							<height>80</height>
						</size>
					</property>
					<property name="maximumSize">
						<size>
							<width>16777215</width>
							<height>80</height>
						</size>
					</property>
					<property name="styleSheet">
						<string notr="true"></string>
					</property>
					<layout class="QVBoxLayout" name="header_layout">
						<property name="spacing">
							<number>2</number>
						</property>
						<property name="leftMargin">
							<number>15</number>
						</property>
						<property name="topMargin">
							<number>10</number>
						</property>
						<property name="rightMargin">
							<number>15</number>
						</property>
						<property name="bottomMargin">
							<number>10</number>
						</property>
						<item>
							<layout class="QHBoxLayout" name="top_header_layout">
								<property name="spacing">
									<number>10</number>
								</property>
								<item>
									<widget class="QLabel" name="title_label">
										<property name="font">
											<font>
												<family>Arial</family>
												<pointsize>14</pointsize>
												<weight>75</weight>
												<bold>true</bold>
											</font>
										</property>
										<property name="styleSheet">
											<string notr="true"></string>
										</property>
										<property name="text">
											<string>UKSDT</string>
										</property>
									</widget>
								</item>
								<item>
									<spacer name="header_spacer">
										<property name="orientation">
											<enum>Qt::Horizontal</enum>
										</property>
										<property name="sizeHint" stdset="0">
											<size>
												<width>40</width>
												<height>20</height>
											</size>
										</property>
									</spacer>
								</item>
								<item>
									<widget class="QPushButton" name="toggle_btn">
										<property name="sizePolicy">
											<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
												<horstretch>0</horstretch>
												<verstretch>0</verstretch>
											</sizepolicy>
										</property>
										<property name="minimumSize">
											<size>
												<width>30</width>
												<height>30</height>
											</size>
										</property>
										<property name="maximumSize">
											<size>
												<width>30</width>
												<height>30</height>
											</size>
										</property>
										<property name="font">
											<font>
												<weight>75</weight>
												<bold>true</bold>
											</font>
										</property>
										<property name="cursor">
											<cursorShape>PointingHandCursor</cursorShape>
										</property>
										<property name="styleSheet">
											<string notr="true"></string>
										</property>
										<property name="text">
											<string>◀</string>
										</property>
									</widget>
								</item>
							</layout>
						</item>
						<item>
							<widget class="QLabel" name="subtitle_label">
								<property name="font">
									<font>
										<family>Arial</family>
										<pointsize>8</pointsize>
									</font>
								</property>
								<property name="styleSheet">
									<string notr="true"></string>
								</property>
								<property name="text">
									<string>Universal KitchenSolution Dev. Tools</string>
								</property>
							</widget>
						</item>
					</layout>
				</widget>
			</item>
			<item>
				<widget class="Line" name="separator">
					<property name="styleSheet">
						<string notr="true"></string>
					</property>
					<property name="lineWidth">
						<number>1</number>
					</property>
					<property name="orientation">
						<enum>Qt::Horizontal</enum>
					</property>
				</widget>
			</item>
			<item>
				<widget class="QScrollArea" name="scroll_area">
					<property name="widgetResizable">
						<bool>true</bool>
					</property>
					<property name="horizontalScrollBarPolicy">
						<enum>Qt::ScrollBarAlwaysOff</enum>
					</property>
					<widget class="QWidget" name="tools_widget">
						<property name="styleSheet">
							<string>background-color:rgb(82, 82, 82);</string>
						</property>
						<property name="geometry">
							<rect>
								<x>0</x>
								<y>0</y>
								<width>278</width>
								<height>617</height>
							</rect>
						</property>
						<layout class="QVBoxLayout" name="tools_layout">
							<property name="spacing">
								<number>8</number>
							</property>
							<property name="leftMargin">
								<number>10</number>
							</property>
							<property name="topMargin">
								<number>10</number>
							</property>
							<property name="rightMargin">
								<number>10</number>
							</property>
							<property name="bottomMargin">
								<number>10</number>
							</property>
							<item>
								<spacer name="tools_spacer">
									<property name="orientation">
										<enum>Qt::Vertical</enum>
									</property>
									<property name="sizeHint" stdset="0">
										<size>
											<width>20</width>
											<height>40</height>
										</size>
									</property>
								</spacer>
							</item>
						</layout>
					</widget>
				</widget>
			</item>
			<item>
				<widget class="Line" name="settings_separator">
					<property name="styleSheet">
						<string notr="true"></string>
					</property>
					<property name="lineWidth">
						<number>1</number>
					</property>
					<property name="orientation">
						<enum>Qt::Horizontal</enum>
					</property>
				</widget>
			</item>
			<item>
				<widget class="QWidget" name="settings_widget" native="true">
					<property name="sizePolicy">
						<sizepolicy hsizetype="Preferred" vsizetype="Fixed">
							<horstretch>0</horstretch>
							<verstretch>0</verstretch>
						</sizepolicy>
					</property>
					<property name="minimumSize">
						<size>
							<width>0</width>
							<height>60</height>
						</size>
					</property>
					<property name="maximumSize">
						<size>
							<width>16777215</width>
							<height>60</height>
						</size>
					</property>
					<property name="styleSheet">
						<string notr="true"></string>
					</property>
					<layout class="QHBoxLayout" name="settings_layout">
						<property name="spacing">
							<number>10</number>
						</property>
						<property name="leftMargin">
							<number>15</number>
						</property>
						<property name="topMargin">
							<number>10</number>
						</property>
						<property name="rightMargin">
							<number>15</number>
						</property>
						<property name="bottomMargin">
							<number>10</number>
						</property>
						<item>
							<spacer name="settings_spacer">
								<property name="orientation">
									<enum>Qt::Horizontal</enum>
								</property>
								<property name="sizeHint" stdset="0">
									<size>
										<width>40</width>
										<height>20</height>
									</size>
								</property>
							</spacer>
						</item>
						<item>
							<widget class="QPushButton" name="settings_btn">
								<property name="sizePolicy">
									<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
										<horstretch>0</horstretch>
										<verstretch>0</verstretch>
									</sizepolicy>
								</property>
								<property name="minimumSize">
									<size>
										<width>40</width>
										<height>40</height>
									</size>
								</property>
								<property name="maximumSize">
									<size>
										<width>40</width>
										<height>40</height>
									</size>
								</property>
								<property name="font">
									<font>
										<family>Arial</family>
										<pointsize>16</pointsize>
									</font>
								</property>
								<property name="cursor">
									<cursorShape>PointingHandCursor</cursorShape>
								</property>
								<property name="toolTip">
									<string>설정</string>
								</property>
								<property name="styleSheet">
									<string notr="true"></string>
								</property>
								<property name="text">
									<string>⚙</string>
								</property>
							</widget>
						</item>
					</layout>
				</widget>
			</item>
		</layout>
	</widget>
	<resources />
	<connections />
</ui>